bundles/core/org.eclipse.smarthome.core/src/main/java/org/eclipse/smarthome/core/items/ItemRegistry.java

```diff
@@ -25,6 +25,14 @@
 public interface ItemRegistry extends Registry<Item, String> {
 
     /**
+     * This methods checks if an item is part of the registry.
+     *
+     * @param name the item name
+     * @return true if there is an item with given name, otherwise false
+     */
+    public boolean containsItem(String name);
```

Comment:
The interface is okay, this implementation is another topic...
Don't need to discuss it here, not related to this changes.
But if you have a look at ItemRegistryImpl you see why I am are not comfortable with the current implementation (`get(itemName)` is using `getItem(itemName)` and catches the exception. `getItem` calls `getItems` that uses `getAll` that creates an copy of...).